src/client/components/Reviews.jsx

```diff
@@ -4,13 +4,25 @@ import Filters from './Filters';
 import List from './List';
 import Pages from './Pages';
 
-const Reviews = () => (
-  <div id="reviews">
-    <Summary />
-    <Filters />
-    <List />
-    <Pages />
-  </div>
-);
+class Reviews extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviewsToDisplay: [{ _id: 12345, rating: 5 }],
+    };
+  }
+
+  render() {
+    const { reviewsToDisplay } = this.state;
+    return (
+      <div id="reviews">
+        <Summary />
+        <Filters />
+        <List reviewsToDisplay={reviewsToDisplay} />
+        <Pages />
```

Comment:
Hey @ChrisALarson, just curious. What was your plan on handling the changes for pages?  Would Pages also need to be a stateful component since you are rendering a new list of reviews? Or are you planning on just limiting each page to a certain number of reviews and essentially rendering those ones once a button is clicked?